src/styles/blocks/product.scss

```diff
@@ -0,0 +1,120 @@
+.product-card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: $product-card-width;
+
+  border: 1px solid $product-border-color;
+  border-radius: $product-border-radius;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+
+    font-family: $product-font-family, sans-serif;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $product-main-color;
+    margin: 0;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $product-seconday-color;
+
+    margin: 4px 0 0;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    text-decoration: none;
+    text-transform: uppercase;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    margin: 16px 0 0;
+
+    background-color: $product-buy-btn-bg-color;
+
+    color: $product-buy-btn-color;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    width: 166px;
+    height: 40px;
+
+    border-radius: $product-buy-btn-border-radius;
+
+    &:hover {
+      color: $product-buy-btn-hover-color;
+      background-color: $product-buy-btn-hover-bg-color;
+
+      border: 1px solid $product-buy-btn-hover-border-color;
+
+      padding: 12px 69px;
+    }
+  }
+}
+
+.product-rating {
+  margin: 16px 0 0;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $product-main-color;
+
+    margin: 0;
+  }
+}
+
+.product-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin: 24px 0 0;
+
+  &__text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $product-seconday-color;
```

Comment:
There is a typo in the variable name `$product-seconday-color`. It should be `$product-secondary-color` to maintain consistency and avoid potential errors.